WordPress/Classes/ViewRelated/Post/EditImageDetailsViewController.m

```diff
@@ -525,9 +525,9 @@ - (void)showAlignmentSelector
 
     PostSettingsSelectionViewController *vc = [[PostSettingsSelectionViewController alloc] initWithDictionary:dict];
     __weak PostSettingsSelectionViewController *weakVc = vc;
-    vc.onItemSelected = ^(NSString *status) {
+    vc.onItemSelected = ^(NSObject *status) {
         // do interesting work here... like updating the value of image meta.
-        self.imageDetails.align = status;
+        self.imageDetails.align = (NSString *)status;
```

Comment:
Status will always be a string in this case, but `PostSettingsSelectionViewController` can handle different types of values. Maybe instead of specifying an `NSObject` in the callback block param, we use `id`.  Looks like we can remove a bunch of the casting this way.